### PR TITLE
add Maternal Death Linkages

### PR DESCRIPTION
**Pull request by saulakravitz**
See https://jira.hl7.org/browse/FHIR-43626
Also added to sandbox

### CODE REVIEW

File: input/fsh/SD_RecordIdentifierFetalDeath.fsh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* component[{year}].value[x] 1..1
+* component[{year}].value[x] only dateTime
+* component[{year}].value[x] ^comment = "The record year is expressed using the YYYY portion of date."
+
+
+Profile: FetalDeathRecordIdentifier
+Parent: Observation
+Id: vrdr-fetal-death-record-identifier
+Title: "Fetal Death Record Identifier"
+Description: "Fetal Death Record Identifier (Observation).  This includes the record identifier, the jurisdiction, and the birth year."
```
Comment (ptran265) [inline input/fsh/SD_RecordIdentifierFetalDeath.fsh:32]: Can we revise this to mirror the BirthRecordIdentifierChild' description so it reads: "Fetal Death Record Identifier (Observation). **For use in Maternal Death.** This includes the record identifier, the jurisdiction, and the birth year."

File: input/fsh/SD_BirthRecordIdentifierInfant.fsh
Comment (ptran265) [inline input/fsh/SD_BirthRecordIdentifierInfant.fsh:1]: Does it make sense to revise the profile name and ID to BirthRecordIdentifierInfant to further distinguish between BirthRecordIdentifierChild? Or would that cause too much hassle for the software folks?

Comment (ptran265) [reply input/fsh/SD_BirthRecordIdentifierInfant.fsh:1]: Or... could BirthRecordIdentifierChild be renamed to ChildsBirthRecordIdentifier? I feel like that name might put more emphasis on the linkage to the mother and better distinguish between the two similar birth record profiles? Without reading the change log and your email, I was a bit confused looking at deathcertificate and these different profiles.

**Review by ptran265 — APPROVED** [review]
Looks good. I left some comments for consideration

**Comment by ptran265** [comment]
Sushi config also needs the two new profiles added

File: input/fsh/SD_BirthRecordIdentifierInfant.fsh
Comment (saulakravitz) [reply input/fsh/SD_BirthRecordIdentifierInfant.fsh:1]: BirthRecordIdentifier ==> BirthRecordIdentifierInfant would require software changes.

Comment (saulakravitz) [reply input/fsh/SD_BirthRecordIdentifierInfant.fsh:1]: I like keeping all of the BirthRecordIdentifier<X> with the <X> as a suffix.  That way the naturally group together.

**Pull request merged**